package.json

```diff
@@ -1,24 +1,46 @@
 {
   "name": "nest-sendgrid",
   "version": "1.0.1",
+  "description": "Lib para envios de emails com sendgrid",
   "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/sthima/sigep.git"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "yarn lint"
+    }
+  },
   "scripts": {
-    "build": "tsc -p tsconfig.lib.json"
+    "start": "ts-node-dev -r tsconfig-paths/register --respawn --transpile-only --ignore-watch node_modules src/index.ts",
```

Comment:
ts-node-dev é mais rapido que o nodemon, coloquei aqui mas ainda não sei um jeito simples de testar um modulo nest, to com uma impressão que preciso fazer uma casca do nest e colocar esse modulo como um import, nao sei, vou buscar depois.